homeassistant/components/transmission/sensor.py

```diff
@@ -82,6 +91,12 @@ def available(self):
         """Could the device be accessed during the last update call."""
         return self._transmission_api.available
 
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes, if any."""
+        if self._torrent_info:
+            return {STATE_ATTR_TORRENT_INFO: self._torrent_info}
+
```

Comment:
`return None` in the other case.

Reply:
I have added that - though it occurs to me that I may have done this wrong in the first place since it looks so different than the other properties, please advise and thank you for your help and patience.

Reply:
How does it look different?

Reply:
I wonder if I should have assigned `{STATE_ATTR_TORRENT_INFO: self._torrent_info}` to a class variable like `self._torrent_info_dict` so that it would be tidier.  Or perhaps it is preferred here that we see it is returning a dictionary and the key and value are explicit?